tensorflow/tools/ci_build/ci_build.sh

```diff
@@ -80,7 +80,7 @@ fi
 
 # cmake (CPU) builds do not require configuration.
 if [[ "${CONTAINER_TYPE}" == "cmake" ]]; then
-  CI_COMMAND_PREFIX=""
+  CI_COMMAND_PREFIX=("")
```

Comment:
The variable was used as an array but is now assigned a string. Bash behaves a little weird here:
https://github.com/koalaman/shellcheck/wiki/SC2178